modules/ting_search/includes/ting_search.admin.inc

```diff
@@ -97,25 +95,6 @@ function ting_search_admin_settings($form_state) {
     '#description' => t('Set the default sorting for search results.'),
   );
 
-  $form['ting_search']['ting_search_result_message'] = array(
-    '#type' => 'item',
-    '#attributes' => array('class' => array('description')),
-    '#title' => t('Display message'),
-    '#markup' => t($display_message),
-    '#description' => t('Message to display when search has more than x results. The token %s will be substituted by the number choosen.'),
-  );
-
-  if ( t($display_message) != $display_message ) {
-    $form['ting_search']['ting_search_result_message']['#description'] .= '<br/>' . t('Translated from: %n.', array('%n' => $display_message));
-  }
-
-  $form['ting_search']['ting_search_result_message_limit'] = array(
```

Comment:
Well I think with your background knowledge, you'd know better if its save to remove

If i just look at the codebase of ding2, it seems safe enough to remove from ding_app_variables, but is it also a variable available as an API? Because then i cant know if any external services uses it

Reply:
I have looked into it, and it seems safe to remove, so i have.
Resolved 👍

Reply:
It don't see any update hook that removes the variable from the database?

Reply:
Good point @cableman - I've added it now https://github.com/ding2/ding2/pull/1164/commits/d926376f7fe1dafba9da56a5d24df2e7a8ad545f#diff-312cde24b21405c928ae5be5da620a03R12 :)